tests/PHPStan/Analyser/data/extends-pdo-statement.php

```diff
@@ -13,6 +13,14 @@ public function setFetchMode($fetchMode, $arg2 = null, $arg3 = null): bool
 	{
 		return true;
 	}
+
+	/**
+	 * @return \Traversable<mixed, mixed>
+	 */
+	public function getIterator()
```

Comment:
So from what i see in the stubs, they already updated `PDOStatement` for `PHP8` which will now implement `IteratorAggregate` not `Traversable` anymore. So i had to add this, since tests for `PHP7` failed.
I am wondering in a real scenario, will the actual implementation of `PDOStatement` be stronger than the stubs?
If not, this could result in "wrong" errors, no?

Reply:
Try adding `@extends IteratorAggregate<...>` here: https://github.com/phpstan/phpstan-src/blob/master/stubs/PDOStatement.stub, these changes you made shouldn't be necessary.

Reply:
Adjusted, but this results in the same it seems as i had before :cry:

Reply:
I don't know how the stub you mentioned is used, but i think the main culprit [is this](https://github.com/JetBrains/phpstorm-stubs/compare/d4db4b4834d8a9c8fc3c0aa6dc3c9bae723bbc9c...874def36f96dbb7db2eb2ff5e47ff54ac3ea34fc#diff-29e016e40b582ba5d0ec160ebbf04f49R1183)
But again, i don't know too much of the stan internals yet